examples/NavigationPlayground/flow-typed/react-navigation.js

```diff
@@ -0,0 +1,1007 @@
+// @flow
+
+import * as React from 'react';
+import { Animated, Image } from 'react-native';
+import type {
```

Comment:
@calebmer, you mentioned on Slack that it shouldn't be possible to import types from package files, and should only be possible to import from other libdefs. However, both this import and the `Layout` import below seem to work. Is this something that will break once I try it on `flow-typed`?

If so, what is the best practice here? The only thing I can think of is to copy-paste the types in, but that seems icky, especially for `StyleObj` which seems to change fairly often. Related: https://github.com/flowtype/flow-typed/issues/631